packages/core/strapi/bin/strapi.js

```diff
@@ -274,18 +273,9 @@ program
   .command('export')
   .description('Export data from Strapi to file')
   .addOption(
-    new Option(
-      '--encrypt <boolean>',
-      `Encrypt output file using the 'aes-128-ecb' algorithm. Prompts for key unless key option is used.`
-    )
-      .default(true)
-      .argParser(parseInputBool)
-  )
-  .addOption(
-    new Option('--compress <boolean>', 'Compress output file using gzip compression')
-      .default(true)
-      .argParser(parseInputBool)
+    new Option('--no-encrypt', `Disable 'aes-128-ecb' encryption of the output file`).default(true)
```

Comment:
Shouldn't it be default false now?

Reply:
Should we make a test for this?

Reply:
We can't test it properly yet, that's what [this PR](https://github.com/strapi/strapi/pull/14637) is for, but we don't have time to get it in before the beta. I may change the target from `main` to `features/deits` to get it in sooner and be able to start writing tests.

Reply:
> It's weird commander logic, --no- is a special prefix that treats it as if it's the value without "no-" but the default is for the value `encrypt` itself

That's so counter-intuitive lol, thanks for the information about the special prefix :slightly_smiling_face: